src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -285,80 +307,150 @@ function Setup({ origin }: { origin: KeylessBackupOrigin }) {
 
   const onPressManualOnboarding = () => {
     ValoraAnalytics.track(KeylessBackupEvents.cab_progress_failed_manual, { origin })
-    navigate(Screens.AccountKeyEducation)
+    isOnboarding
```

Comment:
This also should only be called from onboarding so shouldn't need this check either.

Reply:
Fixed in f797fdfcf188e439b71e6ec58f1eb987e9bfe8bc!